apps/AppWithWearable/flavorizr.yaml

```diff
@@ -0,0 +1,21 @@
+flavors:
```

Comment:
![CleanShot 2024-06-20 at 20 09 55@2x](https://github.com/BasedHardware/Friend/assets/18078879/a40eab62-4e49-4488-8137-a6d472dd64d9)

All versions should have ios 14

Reply:
Please include .dev.env to gitignore

Reply:
Also, please remove default Runner schema. Besides that lgtm, please DM me your transfer details.

Reply:
One last thing you missed probably for android the strings.xml for the app name, please try running it in android.
![CleanShot 2024-06-20 at 20 25 48@2x](https://github.com/BasedHardware/Friend/assets/18078879/026300b6-1660-4c88-9960-8f31244c2e5e)